package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@primer/react",
-  "version": "35.4.0",
```

Comment:
not sure why package-lock doens't update when new versions are released?

Reply:
🤔